docsutils/cli.go

```diff
@@ -22,11 +24,58 @@ const (
 	DocsRepo = "solo-docs"
 )
 
+type DocsPRSpec struct {
+	// Repo owner, i.e. "solo-io"
+	Owner          string
+	// Repo, i.e. "solo-projects"
+	Repo           string
+	// Release tag, i.e. "v0.8.5"
+	Tag            string
+	// Product in docs, i.e. "gloo"
+	Product        string
+	// Project, i.e. "glooe"
+	Project        string
```

Comment:
It's not clear to me what `Project` refers to, whether it is the solo-io/gloo/**project**/<project_name> or something else
`getChangelogFile` seems to make most use of this value:
`// getChangelogFile(gloo, glooe) -> solo-docs/gloo/changelog/glooe-changelog`

The product vs project distinction is unclear to me. glooe seems more like a "product" and gloo more like a "project".

Clarification could be resolved with another pr (or our collective experience with and emergent use of these terms).

Reply:
I agree, I actually incorrectly wrote the comment above the first time because it confused me. I will clean this up a bit before merging.

Reply:
cool, thanks for updating, that makes it very clear